pallets/phala/src/stakepool.rs

```diff
@@ -344,7 +346,9 @@ pub mod pallet {
 		/// The worker doesn't have a valid benchmark when adding to the pool
 		BenchmarkMissing,
 		/// The worker is already added to the pool
-		WorkerExists,
+		WorkerExist,
```

Comment:
Note that, it was `WorkerExists`, but now it's `WorkerExist`. Please carefully check the diff to minimize the changes to the original code.